### PR TITLE
Some improvment

### PR DESCRIPTION
**Pull request by iricge**
hye this is html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js" defer></script>
+        <title>Document</title>
+    </head>
+    <body>
+        <!-- IRIC CODE -->
+        <h1>Hello my freind accept</h1>
```
Comment (taBracket) [inline index.html:13]: you could do it better

File: index.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js" defer></script>
+        <title>Document</title>
```
Comment (taBracket) [inline index.html:9]: you didnt changed title lashi

File: index.html [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+    </head>
+    <body>
+        <!-- IRIC CODE -->
```
Comment (taBracket) [inline index.html:12]: not good

**Review by taBracket — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (iricge) [reply index.html:9]: hamini ke hast

**Pull request merged**